src/TableauConnector/odfe_sql_odbc/manifest.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='odfe_sql_odbc' superclass='odbc' plugin-version='0.2.0' name='ODFE SQL ODBC' version='18.1' min-version-tableau='2020.1'>
+<connector-plugin class='odfe_sql_odbc' superclass='odbc' plugin-version='1.8.0.0' name='SQL' version='18.1' min-version-tableau='2020.1'>
   <vendor-information>
-      <company name="Amazon"/>
+      <company name="Open Distro for ES"/>
       <support-link url="https://github.com/opendistro-for-elasticsearch/sql-odbc"/>
       <driver-download-link url="https://github.com/opendistro-for-elasticsearch/sql-odbc"/>
   </vendor-information>
   <connection-customization class="odfe_sql_odbc" enabled="true" version="10.0">
-    <vendor name="Amazon"/>
-    <driver name="ODFE SQL ODBC Driver"/>
+    <vendor name="Open Distro for ES"/>
```

Comment:
Any specific reason for the name ES rather than Elasticsearch like other places? Or shall we keep it consistent?

Reply:
There is a limit of 24 characters for the Vendor name, which "Open Distro for Elasticsearch" exceeds.

Had discussed this through other means with @anirudha.

Reply:
thanks